public/index.html

```diff
@@ -13,14 +13,17 @@
       <img src="assets/cleanbeach1_img.PNG" class="beach1">
       <img src="assets/cleanbeach2_img.PNG" class="beach2">
     </div>
-      <div class="tools">
+    <div class="tools">
         <img src="assets/InsectNet_icon.png" class="tools0" onclick="toggleInsectNet()">
```

Comment:
```suggestion
        <img src="assets/InsectNet_icon.png" class="insect-net" onclick="toggleInsectNet()">
```

可読性向上の為、命名は説明的にしましょう。